README.md

```diff
@@ -147,6 +147,12 @@ Options:
   -h, --help          Show this message and exit.
 ```
 
+To create a build directory of your local Kedro-Viz instance with static data from the command line, use the following command from the root folder of your Kedro project:
+
+```bash
+kedro viz build
+```
+
```

Comment:
Should we have some docs explaining how to view this build?

Reply:
yes, in another PR we will focus on the build docs